schema/customerror/HttpError.go

```diff
@@ -33,7 +33,11 @@ type HTTPError struct {
 }
 
 func (htErr HTTPError) Error() string {
-	buf := new(bytes.Buffer)
-	buf.ReadFrom(htErr.Body)
-	return fmt.Sprintf("HTTP error %d: %s %s", htErr.StatusCode, htErr.HtErr, buf.String())
+	msg := fmt.Sprintf("HTTP error %d: %s.", htErr.StatusCode, htErr.HtErr)
+	if htErr.Body != nil {
```

Comment:
If body is too large. Please check what would be its impact on err string.

Reply:
We are not using this internally anymore.

Reply:
This is still pending ? Body I assume will be nil with new changes.